cypress/component/forms/ltft/LtftFormView.cy.tsx

```diff
@@ -141,9 +141,6 @@ describe("LTFT Form View - unsubmitted", () => {
     cy.get('[data-cy="ltftName"]').contains("my Unsubmitted LTFT");
     cy.get('[data-cy="ltftCreated"]').should("exist");
     cy.get('[data-cy="ltftModified"]').should("exist");
-    cy.get('[data-cy="ltftModifiedBy"]').contains(
```

Comment:
@EdwardBarclay I think you can restore these deleted tests too so we're testing this bit of the implementation. e.g for this test file, something along these lines:

```
    cy.get('[data-cy="ltftModifiedBy"]').contains("TIS Admin");
```